src/routing_membrane.rs

```diff
@@ -55,6 +55,8 @@ use node_interface::MethodCall;
 use refresh_accumulator::RefreshAccumulator;
 use id::Id;
 use public_id::PublicId;
+use sentinel::pure_sentinel::*;
+use sentinel_request::SentinelPutRequest;
```

Comment:
I would prefer not to use the wildcards, it makes it hard to see where types come from.